cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -346,25 +362,24 @@ describe('Data Table', () => {
 
     const headCellsTooltips = [false, false, true, true, true];
 
-    headCellsTooltips.forEach((isVisible, index) => {
-      const assertion = !isVisible ? 'not.exist' : 'exist';
+    headCellsTooltips.forEach((existsAssertion, index) => {
+      const assertion = !existsAssertion ? 'not.exist' : 'exist';
```

Comment:
```suggestion
    headCellsTooltips.forEach((isVisible, index) => {
      const assertion = !isVisible ? 'not.exist' : 'exist';
```

Reply:
@evansito @josecarlosgonzalezv `is` prefixed names are standard for boolean values like this.
`isVisible true`
is more readable than
`existsAssertion true`